pkg/integrations/github/snippet-linux-ppc64le.sh

```
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+git clone {{ .RunnerImageRepo }} /opt/action-runner-image-pz
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)